includes/utilities.inc

```diff
@@ -462,11 +462,19 @@ function islandora_solr_islandora_basic_collection_backend_callable($collection_
   $qp = new IslandoraSolrQueryProcessor();
   $_islandora_solr_queryclass = $qp;
 
-  $qp->buildQuery(format_string('!member_field:("info:fedora/!pid" OR "!pid") OR !collection_member_field:("info:fedora/!pid" OR "!pid")', array(
-    '!member_field' => variable_get('islandora_solr_member_of_field', 'RELS_EXT_isMemberOf_uri_ms'),
-    '!pid' => $collection_object->id,
-    '!collection_member_field' => variable_get('islandora_solr_member_of_collection_field', 'RELS_EXT_isMemberOfCollection_uri_ms'),
-  )), drupal_get_query_parameters());
+  $qp->buildQuery("*:*", drupal_get_query_parameters());
```

Comment:
Just had a thought (looking at the Solr views code it's the same there), should this be https://github.com/Islandora/islandora_solr_search/blob/7.x/includes/admin.inc#L595?

Reply:
Sounds good to me.